pkg/manager/manager.go

```diff
@@ -92,13 +91,13 @@ func (k *KubeMacPoolManager) Run(rangeStart, rangeEnd net.HardwareAddr) error {
 
 		err = kubevirt_api.AddToScheme(k.runtimeManager.GetScheme())
 		if err != nil {
-			errors.Wrap(err, "unable to register kubevirt scheme")
+			return errors.Wrap(err, "unable to register kubevirt scheme")
```

Comment:
This changes behaviour and it will end `Run` are we sure we want that ?

Reply:
it does change behavior, but I think the former behavior was a bug. 
I mean, how can kubemacpool work properly if kubevirt scheme is not added? or if pool is not allocated correctly?

still it's a change so let's consult with @phoracek